content/overview/pricing/pricing-model.md

```diff
@@ -22,10 +22,12 @@ The current cloud.gov pricing model has three components:
 Find out more about each component via our [pricing terminology page]({{< relref "overview/terminology/pricing-terminology.md" >}}) or see the [dedicated page with our rates]({{< relref "overview/pricing/rates.md" >}}).
 
 ---
-Our mission is to catapult government agencies past mere cloud 'adoption', to the point where they routinely deliver services to the public as fast as they can develop them, all while applying best practices in security and compliance with minimal effort.
+Our mission is broader than making it easier for government agencies to adopt cloud deployment. The mission of cloud.gov is to enable agencies to deliver services to the public as fast as they can develop them while applying best practices in security and compliance with minimal effort.
 
-Comparing cloud.gov's pricing model to private Platform as a Service (PaaS) providers is extremely difficult, as it is not an apples to apples comparison. If you attempt to do a price comparison, be sure to include the up-front and opportunity costs of your procurement, operations, security, and compliance teams in obtaining a solution from scratch. It may be difficult to obtain these costs, especially if they are hidden within various working capital funds in your agency. Consider the labor and time to be saved by having a proven, ready-to-ATO infrastructure in your teams' hands from Day One, via a relatively simple Inter-agency Agreement (IAA). Cloud.gov's resource usage is already priced competitively with the private market, and is the superior option in reducing compliance friction to access the cloud.
+Comparing cloud.gov’s pricing model to private Platform as a Service (PaaS) providers is difficult, as it is not an apples to apples comparison. A price comparison would include the up-front and opportunity costs of procurement, operations, security, and compliance teams in obtaining a solution. These costs can be difficult to isolate, especially if they are covered by various working capital funds in your agency.
 
-We are validating whether our pricing model fits the needs of federal agencies, and will continue to experiment with the model. Our promise to our partners is that prices will not be changed on them mid-fiscal year, and the price of any particular part of the platform will *not go up*.
+cloud.gov avoids many of these costs and delivers a proven, ready-to-ATO infrastructure to your teams’ hands on day one via a relatively simple Inter-agency Agreement (IAA). cloud.gov’s resource usage is already priced competitively with the private market, and is the superior option in reducing compliance friction to access the cloud.
```

Comment:
suggest "via an Inter-agency Agreement (IAA)" since I'm not sure we can promise (yet) that the IAA is relatively simple